packages/figma-utils/src/utils/generate.ts

```diff
@@ -1,40 +1,24 @@
 import { ParsedVariable } from "../types/figma.js";
 
-type GenericGeneratorOptions = {
-  data: ParsedVariable;
-  /**
-   * Function which returns the full variable name depending on the used format.
-   *
-   * @example
-   * ```ts
-   * // for CSS:
-   * (name) => `--${name}`
-   * ```
-   */
-  nameTransformer: (name: string) => string;
-  /**
-   * Function which returns the full variable value if its an alias / reference to another variable.
-   *
-   * @example
-   * ```ts
-   * // for CSS:
-   * (name) => `var(--${name})`
-   * ```
-   */
-  aliasTransformer: (name: string) => string;
-};
-
 /**
  * Generates the given parsed Figma variables into CSS variables.
  *
+ * @param selector CSS selector to use. The mode name will be added to the selector
```

Comment:
doesn't JSDoc want to have all params annotated -> if you annotate `selector`, you should do the same for `data`? (either all or none)

Reply:
Technically you can only document some props but I added the second one to have it complete